plugins/tiddlywiki/markdown/wrapper.js

```diff
@@ -38,13 +38,15 @@ function transformNode(node) {
 		}
 		widget.children = transformNodes(node.slice(p++));
 		// Respect code sections
+		if(widget.tag === "pre") {
+			if (node[1][0]==="code") {
```

Comment:
Maybe we should also check whether the `<code>` tag is the only child of the `<pre>` tag?

Reply:
Actually I just spent a while trying to produce a <pre> that contains anything other than just one <code> - I don't think the markdown-js parser does it. In what case would we get this? User input is first escaped.